app/components/Send/SendPanel/ActivityLink/ActivityLink.jsx

```diff
@@ -0,0 +1,23 @@
+// @flow
+import React from 'react'
+import { NavLink } from 'react-router-dom'
+
+import { ROUTES } from '../../../../core/constants'
+import styles from './ActivityLink.scss'
+
+type Props = {
+  error: boolean,
+}
+
+export const ActivityLink = ({ error }: Props) => (
+  <div>
+    {error
```

Comment:
error && 'your sentence' vs error ? 'your sentence' : ' ', unless you need that space if there is no error